src/random/system.cr

```diff
@@ -0,0 +1,17 @@
+require "sys/random"
+
+# Generates random numbers from a secure source of the system.
+#
+# For example `arc4random` is used on OpenBSD, whereas on Linux it uses
+# `getrandom` (if the kernel supports it) and fallbacks on reading from
+# `/dev/urandom` on UNIX systems.
+class Random::System
+  include Random
```

Comment:
Why not this:
```cr
module Random::System
  extend Random
```
I think it would look awesome!

There is no point in creating an instance of an object without state.